src/components/CurrentWallet.jsx

```diff
@@ -50,24 +48,17 @@ export default function CurrentWallet({ currentWallet }) {
     Promise.all([loadingWallet, loadingUtxos]).finally(() => setIsLoading(false))
 
     return () => abortCtrl.abort()
-  }, [currentWallet])
+  }, [currentWallet, setWalletInfo])
```

Comment:
ESLint wanted me to add this as a dependency (which make sense I guess). I'm not quite sure if I need to wrap it in a `useCallback()` hook to memoize it, though. 

It doesn't cause endless re-renders (I checked that) but I also couldn't find out if the `useContext()` hook (which `setWalletInfo` comes from, see [here](https://github.com/joinmarket-webui/joinmarket-webui/pull/29/files#diff-95bcf45bfc1b1047b24c7448e4036322edfe04e6bcaec7838b986472954725a6R40)) guarantees a stable identity of its return value across re-renders. It seems like it does (otherwise endless re-renders would occur) but the docs didn't mention that.

@dennisreimann if you know of any standard way of handling that, please let me know! 🙏